subsystem/metrics_test.go

```diff
@@ -94,19 +94,20 @@ func getValidationMetricCounter(clusterID strfmt.UUID, validationID models.HostV
 	Expect(err).NotTo(HaveOccurred())
 
 	metrics := strings.Split(string(output), "\n")
-	filteredMetrics := filterMetrics(metrics, string(clusterID), expectedMetric, string(validationID))
+	filteredMetrics := filterMetrics(metrics, expectedMetric, string(validationID))
 	Expect(len(filteredMetrics)).To(Equal(1))
 
 	counter, err := strconv.Atoi(strings.ReplaceAll((strings.Split(filteredMetrics[0], "}")[1]), " ", ""))
 	Expect(err).NotTo(HaveOccurred())
 	return counter
 }
 
-func assertValidationMetricCounter(clusterID strfmt.UUID, validationID models.HostValidationID, expectedMetric string, expectedCounter int) {
-
-	counter := getValidationMetricCounter(clusterID, validationID, expectedMetric)
-	Expect(counter).To(Equal(expectedCounter))
-}
+//TODO Yoni will fix it
```

Comment:
Once we removed the clusterID from the metric, we won't be able to differentiate between different tests - we won't be able to tell what metic was generated from which test and that means all those tests will be reduced to a single test that verify a single validation (which is not too bad since the code is generic for all validations besides `connected` validation).

Reply:
maybe we can verify that the counter was increased and not the exact value

Reply:
This will require to make sure we now exactly what validation each test raises (instead of just knowing "at leas" which validations are changing) - requires to much effort IMO since it doesn't add coverage .